src/store/index.ts

```diff
@@ -39,6 +39,8 @@ export function configureStore(initialState = DEFAULT_STATE) {
       future: [],
       _latestUnfiltered: null,
       group: null,
+      index: null,
+      limit: 30
```

Comment:
Isn't it weird that we have to modify this twice?  Why do we need `initialStateWithHistory` at all? 

Isn't `initialState` sufficient?